lib/galaxy/config/sample/auth_conf.xml.sample

```diff
@@ -102,8 +102,8 @@
             <!-- For Active Directory: -->
 <!--        <bind-user>{sAMAccountName}@dc1.example.com</bind-user>
             <bind-password>{password}</bind-password>
-            <auto-register-username>{sAMAccountName}</auto-register-username>
-            <auto-register-email>{mail}</auto-register-email>
+            <auto-register-username>{username}</auto-register-username>
+            <auto-register-email>{email}</auto-register-email>
```

Comment:
Are those changes due to the upgrade to v3.2.0 ? I've been using these values with our Active Directory server for 5 years.

Reply:
I believe so. I was also using those values for a while, never had any issue. Then I upgraded to 3.2.0 and there were no errors showing up after making these changes. The logging that I did also showed in the dictionary, it only had 'email' and 'username' left in it.

Reply:
I've tested on our production Active Directory server with `python-ldap` 3.2.0 on Python 2.7 and 3.6, it works fine except for the fact that on Python3 the returned values are bytes instead of strings.
I've opened a https://github.com/galaxyproject/galaxy/pull/8823 to revert these config changes and fix the Python3 issue.